### ISSUE TITLE
self-supervised training on Imagenet

### ISSUE THREAD

**Issue by ForawardStar** [Res2Net/Res2Net-PretrainedModels#75]
Is it possible to provide the self-supervised pre-trained model? Specifically, the model is trained on Imagenet without using the ground truth label. The model is trained in an self-supervised manner, e.g., momentum contrastive learning (https://github.com/facebookresearch/moco)
